tests/conftest.py

```diff
@@ -1,42 +1,3 @@
-import os
-import pytest
-import httpretty
-from dotenv import load_dotenv
-from fastapi.testclient import TestClient
-
-from src.app import app
-from src.apis.schemas.header_schema import HeaderRequest
-
-
-@pytest.fixture
-def client():
-    """
-    This client can call the developed application
-    """
-
-    with TestClient(app) as test_client:
-        yield test_client
-
-
-@pytest.fixture
-def server():
-    """
-    This server can receive requests and mock the response
-    """
-
-    load_dotenv('default.env')
-
-    baseurl = 'http://testserver'
-    require_headers = HeaderRequest(
-        USERID_HEADER_KEY=os.environ.get('USERID_HEADER_KEY'),
-        GROUPS_HEADER_KEY=os.environ.get('GROUPS_HEADER_KEY'),
-        CLIENTTYPE_HEADER_KEY=os.environ.get('CLIENTTYPE_HEADER_KEY'),
-        BACKOFFICE_HEADER_KEY=os.environ.get('BACKOFFICE_HEADER_KEY'),
-    ).__dict__.items()
-
-    httpretty.enable(verbose=False, allow_net_connect=False)
-
-    yield baseurl, require_headers
-
-    httpretty.disable()
-    httpretty.reset()
+# pylint: disable=W0611
```

Comment:
Do we need this comment or can it be removed?

Reply:
This comment is for the linter to prevent it from reporting an error. Specifically, it reports that among the arguments of a function, parameters that do not have a default value should be declared first. I did not correct the order of the parameters because I am in fact maintaining the same interface used by httpretty's register_uri method